apps/web/src/client/components/file-uploader/_server-actions.js

```diff
@@ -72,9 +74,12 @@ const serverActions = (uploadForm) => {
 	 */
 	const uploadOnBlobStorageUpload = async (fileToUpload, uploadInfo) => {
 		return new Promise((resolve) => {
+			const blobServiceClient = BlobServiceClient;
+			// TODO: connect to the blob storage and upload the file.
+
```

Comment:
This doesn't seem to relate to the rollup changes?

Reply:
no, but the problem with rollup was that I was unable to import this. Added this line as a console.log to verify the fix effectivness and then I changed it to this